web/src/settings_bots.js

```diff
@@ -24,6 +24,7 @@ import * as user_profile from "./user_profile";
 
 const INCOMING_WEBHOOK_BOT_TYPE = 2;
 const OUTGOING_WEBHOOK_BOT_TYPE = "3";
+const OUTGOING_WEBHOOK_BOT_TYPE_INT = 3;
 const EMBEDDED_BOT_TYPE = "4";
```

Comment:
Should these just all be in integer format? I see `INCOMING_WEBHOOK_BOT_TYPE` is already 2. Maybe worth having a comment about what's going on here or just cleaning it up.

Reply:
The string values are used to compare the value selected in "Bot type" dropdown because `$("#create_bot_type").val()` returns string. We can also do `OUTGOING_WEBHOOK_BOT_TYPE.toString()` while comparing.

Reply:
That seems probably nicer. Not a blocker for this PR.